pkg/booleanpolicy/field_metadata.go

```diff
@@ -771,7 +771,7 @@ func initializeFieldMetadata() FieldMetadata {
 		fieldnames.KubeResourceName,
 		querybuilders.ForFieldLabel(augmentedobjs.KubernetesResourceNameCustomTag), nil,
 		func(*validateConfiguration) *regexp.Regexp {
-			return kubernetesNameRegex
+			return stringValueRegex
```

Comment:
Just gotta be careful with what happens if a user changes to regex and then reverts to an older version

Reply:
Yup, this will cause the policy to not be inserted which I think is okay as it would have to be a net new policy added in the most recent version